notebooks/IV-queries.sparqlbook

```diff
@@ -0,0 +1,176 @@
+[
+  {
+    "kind": 1,
+    "language": "markdown",
+    "value": "#### IVGR2: Onderzoek van de geloofsbrieven",
+    "metadata": {}
+  },
+  {
+    "kind": 1,
+    "language": "markdown",
+    "value": "##### IVGR2-LMB-1: De gemeenteraad keurt de geloofsbrieven goed van volgende verkozenen:",
+    "metadata": {}
+  },
+  {
+    "kind": 2,
+    "language": "sparql",
+    "value": "PREFIX org: <http://www.w3.org/ns/org#>\nPREFIX ext: <http://mu.semte.ch/vocabularies/ext/>\nPREFIX lmb: <http://lblod.data.gift/vocabularies/lmb/>\nPREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>\nPREFIX persoon: <http://data.vlaanderen.be/ns/persoon#>\nPREFIX foaf: <http://xmlns.com/foaf/0.1/>\nSELECT DISTINCT ?persoon ?naam WHERE {\n  # TODO: replace this by the correct 'bestuursperiode'\n  ?bestuursorgaan lmb:heeftBestuursperiode <http://data.lblod.info/id/concept/Bestuursperiode/a2b977a3-ce68-4e42-80a6-4397f66fc5ca>.\n  ?bestuursorgaan org:hasPost ?mandaat.\n  ?mandataris org:holds ?mandaat.\n  ?mandataris mandaat:isBestuurlijkeAliasVan ?persoon.\n  ?persoon persoon:gebruikteVoornaam ?voornaam.\n  ?persoon foaf:familyName ?achternaam.\n  BIND(CONCAT(?voornaam, \" \", ?achternaam) AS ?naam)\n} \n# TODO: remove this limit in production\nLIMIT 10",
+    "metadata": {}
+  },
+  {
+    "kind": 1,
+    "language": "markdown",
+    "value": "#### IVGR3: Eedaflegging van de verkozen gemeenteraadsleden ",
+    "metadata": {}
+  },
+  {
+    "kind": 1,
+    "language": "markdown",
+    "value": "##### IVGR3-LMB-1: De gemeenteraad neemt kennis van de eedaflegging van de volgende verkozenen op datum van",
+    "metadata": {}
+  },
+  {
+    "kind": 2,
+    "language": "sparql",
+    "value": "PREFIX org: <http://www.w3.org/ns/org#>\nPREFIX ext: <http://mu.semte.ch/vocabularies/ext/>\nPREFIX lmb: <http://lblod.data.gift/vocabularies/lmb/>\nPREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>\nPREFIX persoon: <http://data.vlaanderen.be/ns/persoon#>\nPREFIX foaf: <http://xmlns.com/foaf/0.1/>\nSELECT DISTINCT ?persoon ?naam ?datum_eedaflegging WHERE {\n  # TODO: replace this by the correct 'bestuursperiode'\n  ?bestuursorgaan lmb:heeftBestuursperiode <http://data.lblod.info/id/concept/Bestuursperiode/a2b977a3-ce68-4e42-80a6-4397f66fc5ca>.\n  ?bestuursorgaan org:hasPost ?mandaat.\n  ?mandataris org:holds ?mandaat.\n  ?mandataris mandaat:isBestuurlijkeAliasVan ?persoon.\n  ?persoon persoon:gebruikteVoornaam ?voornaam.\n  ?persoon foaf:familyName ?achternaam.\n  BIND(CONCAT(?voornaam, \" \", ?achternaam) AS ?naam)\n  # TODO: replace this by the correct date (start or ending of the inauguration meeting)\n  BIND(NOW() AS ?datum_eedaflegging)\n} \n# TODO: remove this limit in production\nLIMIT 10",
```

Comment:
This will indeed come from the meeting we're creating (essentially the start date). I just added a dummy date for now, but that will not be in the final results. The date will not come from the sparql query.